pages/phases/getting-started/selecting-smes.md

```diff
@@ -27,5 +27,5 @@ It's critical for you to identify a project manager to manage the deliverables r
 - **Resume Review Training:** Before SMEs begin reviewing resumes, HR will conduct a mandatory two-hour resume review training, followed by 1 hour of reviewing actual resumes in their list with HR present to answer questions.
 - **Resume Review:** HR will divide applicants evenly among groups of two SMEs. They will have four days to review all the resumes in their list. Each resume requires two reviews.
 - **Resume Review Tiebreaker:** Over a two-day period, an additional SME will be assigned to each divided list of applicants to break any ties between the first two assessments.
-- **Phone Assessment Interview Training:** While phone interviews are being scheduled with applicants, HR will conduct a mandatory two-hour interview training for all SMEs conducting the first and second phone assessment interviews. This training explains how to conduct structured interviews and assess an applicant’s proficiency with the required competencies based on their responses.
-- **Phone Assessment Interviews:** Over a 2-3 week period, SMEs will contribute up to 20 hours per week conducting interviews with applicants. This assumes each interview is one hour long. This includes a recommended 45 minutes after each interview for completing the interview rating sheet and adding their feedback to the talent acquisition system. The time required will vary depending on how many applicants moved forward past the resume review stage and the first phone interview.
+- **Assessment Training:** While phone interviews or other assessments are being set-up with applicants, HR will conduct a mandatory two-hour training for all SMEs conducting assessments (such as, but not only, interviews). This training explains how to conduct an interview or written assessment review, and assess an applicant’s proficiency with the required competencies based on their responses.
```

Comment:
i dont like "such as but not only interviews" i'd rather say "structured interviews, written assessments, demonstration projects, etc"

Reply:
done